pxc/docker/install-deps

```diff
@@ -63,7 +63,7 @@ if [ -f /usr/bin/yum ]; then
     libzstd-devel tzdata check-devel scons libgcrypt-devel libev-devel zlib-devel libaio-devel \
     ncurses-devel readline-devel pam-devel socat vim-common openssl perl-CPAN rsync libcurl-devel \
     python3 sysbench perl-DBD-mysql \
-    patchelf lsof \
+    patchelf lsof boost-devel \
```

Comment:
Did you test it for Centos7/Centos6 ? If it is needed only for Centos8 please add it only for it.

Reply:
As I can see from the logs, it's only needed for CentOS 8, but Debian family has a `libboost-all-dev` package listed in `PKGLIST`